falcon/media/json.py

```diff
@@ -11,21 +12,70 @@ class JSONHandler(BaseHandler):
 
     This handler uses Python's :py:mod:`json` by default, but will
```

Comment:
This comment about ``ujson`` no longer holds as I understand?

Reply:
[fixed]